src/frontend/js/index.js

```diff
@@ -918,7 +918,7 @@ async function getDomain(dependent_variable, independent_variables = {}, pick =
 
 const flowLineControls = state => {
     return [
-        state.selectedvars.dependent_variable !== "" && state.selectedZones.length == 0 && [m('br'), m('p', "(Click a zone to see incoming flows)")],
+        state.selectedvars.dependent_variable !== '' && state.selectedZones.length === 0 && !state.compare && [m('br'), m('p', '(Click a zone to see outgoing flows)')],
```

Comment:
```suggestion
        state.selectedvars.dependent_variable !== '' && state.selectedZones.length === 0 && !state.compare && [m('br'), m('p', '(Click a zone to see incoming flows)')],
```

I'm assuming this was something missed in the merge :)